### PR TITLE
8688xtzda make user and organization queries more precise

### PR DESCRIPTION
**Pull request by rohanshah**
**ClickUp:** https://app.clickup.com/t/8688xtzda

### CODE REVIEW

File: terraform/pre-sign-up/lambda.py
Comment (jfromme) [inline terraform/pre-sign-up/lambda.py:1]: /

Comment (jfromme) [inline terraform/pre-sign-up/lambda.py:1]: Looks good, but I think there is one more place to update. The `RETURNING *` in this statement at line 142:
```
    statement = f"INSERT INTO pennsieve.users({column_list(user_insert_columns)}) VALUES({value_list(user_insert_values)}) RETURNING *"
```

Comment (rohanshah) [reply terraform/pre-sign-up/lambda.py:1]: ah nice catch ! ty!

**Review by jfromme — APPROVED** [review]
Very nice

**Pull request merged**